Oswaldo Vazquez Aparicio/26 Agosto/miniproject/ShoppingList.java

```diff
@@ -0,0 +1,29 @@
+package miniproject;
+
+import java.util.LinkedList;
+
+public class ShoppingList {
+
+    private final LinkedList<Product> products;
+
+    public ShoppingList(){
+        products = new LinkedList<>();
+    }
+
+    public void addProductToBuy(Product product){
+        products.add(product);
+    }
+
+    public void printProducts(){
+        String tableFormat = "%-30s%-15s%-15s%-15s%n";
```

Comment:
Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings